hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/ipc/TestRPC.java

```diff
@@ -1697,6 +1700,61 @@ public void testRpcMetricsInNanos() throws Exception {
     }
   }
 
+  @Test
+  public void testNumTotalRequestsMetrics() throws Exception {
+    UserGroupInformation ugi = UserGroupInformation.
+        createUserForTesting("userXyz", new String[0]);
+
+    final Server server = setupTestServer(conf, 1);
+
+    ExecutorService executorService = null;
+    try {
+      RpcMetrics rpcMetrics = server.getRpcMetrics();
+      assertEquals(0, rpcMetrics.getTotalRequests());
+      assertEquals(0.0, rpcMetrics.getTotalRequestsPerSecond(), 0.0);
```

Comment:
RpcMetrics#getTotalRequestsPerSecond return `long`, so we can make some change.

```suggestion
      assertEquals(0, rpcMetrics.getTotalRequestsPerSecond());
```